nbconvert/preprocessors/execute.py

```diff
@@ -184,7 +184,7 @@ def run_cell(self, cell):
         while True:
             try:
                 timeout = self.timeout
-                if timeout <= 0:
```

Comment:
I was suggesting that we document what was already happening, rather than change behaviour. If `0` does already work to suspend the timeout, that's fine. I thought it didn't, but I may have been mistaken. I don't see any need for 0 to be instant-timeout, because execution always takes some time, even if it's very small.